core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -49,6 +49,23 @@
           <li class="nav-item"> <a ng-keydown="$ctrl.onMenuKeypress($event, 'aboutMenu', {tab: $ctrl.ACTION_CLOSE})" class="dropdown-item nav-link oppia-navbar-tab-content" href="https://oppiafoundation.org/volunteer" target="_blank" rel="noopener" translate="I18N_TOPNAV_GET_INVOLVED"></a></li>
         </ul>
       </li>
+      <li ng-class="{'open' : $ctrl.activeMenuName === 'aboutMenu'}" uib-dropdown ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_ABOUT" class="nav-item  oppia-navbar-clickable-dropdown protractor-test-about-oppia-list-item">
```

Comment:
You're using `about` a lot here, shouldn't they be changed to be classroom? I.e.

```diff
-  <li ng-class="{'open' : $ctrl.activeMenuName === 'aboutMenu'}" uib-dropdown ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_ABOUT" class="nav-item  oppia-navbar-clickable-dropdown protractor-test-about-oppia-list-item">
+  <li ng-class="{'open' : $ctrl.activeMenuName === 'classroomMenu'}" uib-dropdown ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_CLASSROOM" class="nav-item  oppia-navbar-clickable-dropdown protractor-test-classroom-oppia-list-item">
```

Reply:
Thank you for highlighting, I will change it asap to classroom , has the show_classroom_callout flag been used correctly?

Reply:
I believe that the `<li>` element here needs to have `SHOW_CLASSROOM_CALLOUT` if my understanding is correct that you don't want to show the button when the value of `SHOW_CLASSROOM_CALLOUT` is false.

Reply:
So the value of the SHOW_CLASSROOM_CALLOUT should be initialised to true and the ng if be added to the <li> element?
What does it mean by hiding the option under the 
SHOW_CLASSROOM_CALLOUT flag?

Reply:
Ah, I just meant whether we should be able to toggle the value of `SHOW_CLASSROOM_CALLOUT` between true or false

Reply:
I am sorry, I did not understand 😅

Reply:
Okay, Will look into it and make the changes as mentioned before asap.

Reply:
Awesome! Once you're ready, please feel free to make an official PR to Oppia :)

Reply:
Okay, Will look into it and make the changes as mentioned before asap.

Reply:
Sure @brianrodri , thank you so much for helping out :)